go.mod

```diff
@@ -108,6 +108,8 @@ require (
 	upper.io/db.v3 v3.7.1+incompatible
 )
 
+require github.com/mattn/go-sqlite3 v2.0.2+incompatible
```

Comment:
Please move this line to the first `require` group.

Reply:
Fix it.